makefiles/Makefile.docs-kotlin-sync

```diff
@@ -0,0 +1,21 @@
+GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+
+ifeq ($(STAGING_USERNAME),)
+	USER=$(shell whoami)
+else
+	USER=$(STAGING_USERNAME)
+endif
+
+PROJECT=kotlin-sync-driver
+MUT_PREFIX ?= $(PROJECT)
+
+REPO_DIR=$(shell pwd)
+
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+include ~/shared.mk
+
+
+get-build-dependencies: 
+	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/docs-node.yaml > ${REPO_DIR}/published-branches.yaml
```

Comment:
According to the wiki page, we can make this a no-op

```
get-build-dependencies:
    @echo "Published branches information stored in Atlas collection"
```

Reply:
originally I had that, but I copied over the document entry and it had that URL, so I updated the `Makefile` to reflect that. But I reverted that change and it is a `no-op` again.